core/controllers/suggestion.py

```diff
@@ -50,6 +50,15 @@ def _require_valid_suggestion_and_target_types(target_type, suggestion_type):
             'Invalid suggestion_type: %s' % suggestion_type)
 
 
+def _target_id_to_opportunity_dict(suggestions):
```

Comment:
Always start method/function names with an appropriate verb. That way, you also don't end up with confusion between target_id_to_opportunity_dict (a thing) and _target_id_to_opportunity_dict (a function).